### PR TITLE
fix: Update LabelTo and SendMessage to Interpolated Strings (#1053)

### LINKED ISSUE #1053: Update LabelTo and SendMessage to Interpolated Strings

### PR DESCRIPTION
**Pull request by fabrizioiacobucci**
Closes #1053

### CODE REVIEW

File: Projects/UOContent/Multis/Houses/HouseFoundation.cs [hunk 1]
```diff
@@ -1156,9 +1156,7 @@ public void EndConfirmCommit(Mobile from)
                 if (from.AccessLevel >= AccessLevel.GameMaster && cost != 0)
                 {
                     from.SendMessage(
-                        "{0} gold would have been {1} your bank if you were not a GM.",
-                        cost.ToString(),
-                        cost > 0 ? "withdrawn from" : "deposited into"
+                        $"{cost.ToString()} gold would have been {(cost > 0 ? "withdrawn from" : "deposited into")} your bank if you were not a GM."
```
Comment (kamronbatman) [inline Projects/UOContent/Multis/Houses/HouseFoundation.cs:1159]: Let's remove the `.ToString()` from inside the interpolations. Not only is it syntactically redundant but .net core will use TryFormat instead if the value is a primitive or ISpanFormattable which is much more optimized.

File: Projects/UOContent/Spells/First/NightSight.cs [hunk 2]
```diff
@@ -58,7 +58,7 @@ protected override void OnTarget(Mobile from, object targeted)
                     }
                     else
                     {
-                        from.SendMessage("{0} already have nightsight.", from == targ ? "You" : "They");
+                        from.SendMessage($"{(from == targ ? "You" : "They")} already have nightsight.");
```
Comment (kamronbatman) [inline Projects/UOContent/Spells/First/NightSight.cs:61]: We don't have to do it here but basic conditionals should be changed to if/else with static messages.

File: Projects/UOContent/Items/Skill Items/Magical/Misc/RecallRune.cs [hunk 3]
```diff
@@ -268,40 +268,38 @@ public override void OnSingleClick(Mobile from)
...
                     );
                 }
                 else
                 {
                     LabelTo(
                         from,
-                        House != null ? "{0} ({1})(House)" : "{0} ({1})",
-                        string.Format(RuneFormat, desc),
-                        m_TargetMap
+                        $"{string.Format(RuneFormat, desc)} ({m_TargetMap}){(House != null ? "(House)" : string.Empty)}"
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Skill Items/Magical/Misc/RecallRune.cs:302]: The variable RuneFormat is not a string formatter anymore. Let's flatten this out into the parent interpolation.

**Review by kamronbatman — CHANGES_REQUESTED** [review]

File: Projects/UOContent/Multis/Houses/HouseFoundation.cs (hunk 1 above)
Comment (fabrizioiacobucci) [reply Projects/UOContent/Multis/Houses/HouseFoundation.cs]: Done.
Additional thanks for the inside information

File: Projects/UOContent/Spells/First/NightSight.cs (hunk 2 above)
Comment (fabrizioiacobucci) [reply Projects/UOContent/Spells/First/NightSight.cs]: Done, also for other occurrencies.

File: Projects/UOContent/Items/Skill Items/Magical/Misc/RecallRune.cs (hunk 3 above)
Comment (fabrizioiacobucci) [reply Projects/UOContent/Items/Skill Items/Magical/Misc/RecallRune.cs]: Done.

**Comment by kamronbatman** [comment]
Thank you for the contribution. Please sign the CLA. I'll do another review when I get home tonight or tomorrow then merge.

**Review by kamronbatman — APPROVED** [review]

**Pull request merged**